lighthouse-core/audits/error-logs.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview Audits a page to determine if it is calling deprecated APIs.
+ * This is done by collecting console log messages and filtering them by ones
+ * that contain deprecated API warnings sent by Chrome.
+ */
+
+const Audit = require('./audit');
+
+class ErrorLogs extends Audit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'ErrorLogs',
+      name: 'error-logs',
+      description: 'No browser errors logged to the console',
+      helpText: `
```

Comment:
this ends up causing a bunch of white space in the `helpText`, which isn't a huge deal in the html, but somewhat more annoying if you're consuming the JSON (though to be fair, you still have to deal with markdown in the `helpText`). Either way, for all other audits we're just using string concat, so better to stick to that format:
```js
helpText: 'Errors logged to the console indicate unresolved problems. ' +
    'They can come from network request failures and other browser concerns.';
```